runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewInternal.java

```diff
@@ -1312,4 +1314,12 @@ public void onScrollChangedDelegate(int l, int t, int oldl, int oldt) {
               preWrapperLines = {"onFocusChanged(gainFocus, direction, previouslyFocusedRect);"})
     public void onFocusChangedDelegate(boolean gainFocus, int direction, Rect previouslyFocusedRect) {
     }
+
+    // Override XWalkView.setOnTouchListener to install the listener to ContentView
+    // therefore touch event intercept through onTouchListener is available on XWalkView.
+    @Override
+    @XWalkAPI
+    public void setOnTouchListener(OnTouchListener l) {
```

Comment:
Xing is right. I just copy and paste it from Android SDK doc.
```public void setOnTouchListener (View.OnTouchListener l)```.
Since it's a Android View API and it is always there but just not working, adding @Since xx may cause confusion, and it's not exposed in XWalkView API doc as no javadoc  tag is added here.